autocorpus/Autocorpus.py

```diff
@@ -358,9 +350,27 @@ def __merge_table_data(self):
         else:
             return
 
+    def process_files(self):
+        assert self.config, "A valid config file must be loaded."
```

Comment:
Weirdly in Python if you build a package in optimised mode it strips out `assert` statements, so you can't rely on it for error handling, in case a user has installed it this way.

```suggestion
        if not self.config:
            raise RuntimeError("A valid config file must be loaded.")
```